src/hooks/useWeb3Modal.ts

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react';
+import { ModalCtrl } from '../controllers/ModalCtrl';
+import { ClientCtrl } from '../controllers/ClientCtrl';
+
+export function useWeb3Modal() {
```

Comment:
I see this export + from initial search in VSCode it shows it is exported in index.tsx. Is there specific usage of this used in examples or elsewhere? Apologies if I am missing something

Reply:
the exports in `index.tsx` are the "public" exports from the SDK, that are used in the example app.

`import { Web3Modal, Web3Button, useWeb3Modal } from '@web3modal/react-native';`